wordcloud/wordcloud.py

```diff
@@ -0,0 +1,46 @@
+import os
+from operator import itemgetter
+import re
+
+from haystack.query import SearchQuerySet
+from pombola.hansard import models as hansard_models
+
+
+BASEDIR = os.path.dirname(__file__)
+# normal english stop words and hansard-centric words to ignore
+STOP_WORDS = open(os.path.join(BASEDIR, 'stopwords.txt'), 'rU').read().splitlines()
+
+
+def recent_entries(max_entries=20):
+    return SearchQuerySet().models(hansard_models.Entry).order_by('-sitting_start_date')[:max_entries]
```

Comment:
I copied this from the original code, and as I did it I had the same thought as you. The reasons I kept the ElasticSearch version was that we can use the all the content together thing that's used by search for the words - so whatever model, we can have all the words from all the wordy bits, as it were. (I realize we'd have to refactor a tiny bit to use a different model, but you know what I mean).